templates/mxnet-cu90-1.3.1/requirements.txt

```diff
@@ -0,0 +1,4 @@
+algorithmia>=1.0.0,<2.0
+six
```

Comment:
minor: as of yesterday you don't have to include `six` in here anymore since it will be installed with the pipe script (which is what actually needs it in the first place).